### PR TITLE
ZEPPELIN-1224: Fix typo in method name

### PR DESCRIPTION
**Pull request by karup1990**
### What is this PR for?
Get  the correct method to invoke, for autocompletion

### What type of PR is it?
Bug Fix

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-1224

### How should this be tested?
Try to get the auto complete suggestion.(`cmd + .`)
We should not get `NoSuchMethodError` and suggestions should be listed rightly

### Screenshots (if appropriate)
NA

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java [hunk 1]
```diff
@@ -748,7 +748,7 @@ public void open() {
       cursor = completionText.length();
     }
     if (Utils.isScala2_10()) {
-      ScalaCompleter c = (ScalaCompleter) Utils.invokeMethod(completor, "completor");
+      ScalaCompleter c = (ScalaCompleter) Utils.invokeMethod(completor, "completer");
```
Comment (AhyoungRyu) [inline spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java]: @karup1990 Thanks for fixing this.
Why don't you change another `completor` in this line? I mean
```
ScalaCompleter c = (ScalaCompleter) Utils.invokeMethod(completer, "completer");
```

Then you have to change [spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java#L114](https://github.com/apache/zeppelin/blob/master/spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java#L114) & [spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java#L116](https://github.com/apache/zeppelin/blob/master/spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java#L116) too.

Comment (AhyoungRyu) [inline spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java]: More `completor` in here :)[spark/src/main/java/org/apache/zeppelin/spark/DepInterpreter.java#L79](https://github.com/apache/zeppelin/blob/master/spark/src/main/java/org/apache/zeppelin/spark/DepInterpreter.java#L79) & [spark/src/main/java/org/apache/zeppelin/spark/DepInterpreter.java#L81](https://github.com/apache/zeppelin/blob/master/spark/src/main/java/org/apache/zeppelin/spark/DepInterpreter.java#L81)

**Comment by karup1990** [comment]
I was thinking to keep changes minimal.
Should I change all variable names ? @AhyoungRyu

**Comment by AhyoungRyu** [comment]
I tried to find all `completor` from the root dir. That I found was all. Since it's just 8 part in 2 files (`SparkInterpreter.java` & `DepInterpreter.java`), we don't need to remain typos i think. What do you think? :)

**Comment by karup1990** [comment]
Makes sense. Done

**Comment by AhyoungRyu** [comment]
@karup1990 Thanks! Looks good 👍

**Comment by Leemoonsoo** [comment]
LGTM. 

CI failure irrelevant. Merge it into master and branch-0.6  if there're no more discussions.

**Pull request closed without merging**